pkg/kubelet/cpumanager/cpu_manager.go

```diff
@@ -80,7 +85,7 @@ type manager struct {
 var _ Manager = &manager{}
 
 // NewManager creates new cpu manager based on provided policy
-func NewManager(cpuPolicyName string, cr internalapi.RuntimeService, kletGetter kletGetter, statusProvider status.PodStatusProvider) (Manager, error) {
+func NewManager(cpuPolicyName string, cr internalapi.RuntimeService, kletGetter kletGetter, statusProvider status.PodStatusProvider) (Manager, lifecycle.PodAdmitHandler, error) {
```

Comment:
NOTE: Conflict here is due to a change @balajismaniam made for unit testing. He made a new interface to abstract only the one function we need from the runtime service (UpdateContainerResources) so the mock is smaller.